tools/grub/build.yml

```diff
@@ -0,0 +1,2 @@
+image: grub
+network: true
```

Comment:
do you need to add platforms here (only builds on arm64, amd64)?

Reply:
Right now, only on `arm64` and `amd64`. We could add others in the future, as grub does support it. I don't recall the `platforms` key in `build.yml` and it isn't in the docs?

Reply:
Ah `arches`. No problem. Added.